finality-aleph/src/network/gossip/service.rs

```diff
@@ -330,12 +396,25 @@ mod tests {
     #[async_trait::async_trait]
     impl Network<MockData> for TestData {
         type Error = Error;
+        type PeerId = MockPublicKey;
+
+        fn send(&mut self, data: MockData, peer_id: Self::PeerId) -> Result<(), Self::Error> {
```

Comment:
We should probably add tests for `send` and `send_to_random`. For the latter it would be nice to have one test where peer_ids has intersection of only one peer and another where intersection is zero peers.

Reply:
Added tests, good point.